src/data/posts/twitter-and-the-perils-of-obedience.mdx

```diff
@@ -0,0 +1,172 @@
+/>
+
+<br />
+
+However, the actual results of the experiment revealed a surprising truth about human behavior:
+
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/actual-results"
+  alt="Actual Results: 59.1% completed the full series of shocks, 40.9% refused to continue at some point"
+  caption="The experimental results were much more grim."
+/>
+
+Recreations of this experiment have produced similar results, [as recently as 2009](https://www.apa.org/news/press/releases/2008/12/replicating-milgram).
+
+## Obedience and the Perils of Twitter
+
+I can't help but draw a connection between the Milgram experiment and the current state of Twitter. The experiment was designed to test the willingness of people to obey authority figures. Twitter's recent acquisition by a megalomaniacal celebrity narcissist is an amplified example of this phenomenon in action.
+
+Under Musk's leadership, Twitter's employee base has been slashed from 7,500 to approximately 2,000 in the last month, in an attempt to cut costs after a purchase that only made sense for a rich, vein billionaire who wanted to flex his ego.
+
+I've seen plenty of discussion online that many of twitter's remaining engineers are likely tied to H1B visas, and are unable to leave their job without risking deportation. There is no doubt another contingent of employees who believe in Musk's vision, who are bought into the idea of being an elite engineer working on a desperately slim team, and are willing to work for him despite the fact that he's a complete and utter asshole.
+
+I don't think it is a stretch to say that employees at Twitter are being treated poorly, and have been given increasingly shocking orders from their new CEO (for example, [1](https://twitter.com/ZoeSchiffer/status/1593649356661436417), [2](https://twitter.com/alexeheath/status/1595701492006014977?t=7MMiqyX0-LWIXExz6QtkAQ&s=19)).
+
+Since rumors of Musk's takeover, legions of advertisers have pulled their ad spend from Twitter, for fear of being associated with the company. The company's income has plummeted, and Musk is clearly trying to cut costs wherever he can, while adding features as a part of a desperate money grab to keep the company from going bankrupt.
+
+I left twitter recently, shortly after a hasty user poll resulted in the reinstatement of former (and twice-impeached) president Trump's account.
+
+<Tweet align="center" tweetLink="irreverentmike/status/1594455660040572928" />
+
+Since then, another hasty poll has resulted in an absolutely staggering policy change: all\* accounts that were previously suspended for violating Twitter's rules will be reinstated.
+
+<Tweet align="center" tweetLink="elonmusk/status/1595473875847942146" />
+
+Pandora's box has been opened, and the monsters are coming back. Twitter is now a platform for hate speech, misinformation, and disinformation. It's will become a cesspool of conspiracy theories, and a breeding ground for countless nightmares and real-world dangers.
```

Comment:
Devil's advocate viewpoint ... it's always been a platform for these things, but you could argue Reddit is the same thing. It's only a platform for these things, if you're seeking it out as a user. I see very little of this, and 99% of the time I only see it when someone is denouncing it. It's a perpetual echo chamber if anything, and allows people to feel that their horrible beliefs are shared with millions of people, even if the reality is a small fraction of that.

Reply:
This is a great place to tie in the insurrection though ... the real-world danger is most pertinent to the argument, and was a very tangible event that took place in large part because of the vitriol spewed by the now unblocked orange man.